### PR TITLE
Use an automated PR linter, danger.systems.

### PR DESCRIPTION
**Pull request by dblock**
I'll iterate on this...

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -15,3 +15,5 @@ if RUBY_VERSION < '2.2.2'
   gem 'rack', '<2.0.0'
   gem 'activesupport', '<5.0.0'
 end
+
+gem 'danger', '~> 2.0'
```
Comment (LeFnord) [inline Gemfile]: Maybe adding `required: false`, so not to introduce unwanted dependency?

Comment (dblock) [inline Gemfile]: Good idea!

**Comment by Bugagazavr** [comment]
Looks like danger use redcarpet only,  this not possible with jruby, should be:

```rb
unless RUBY_PLATFORM == 'java'
  gem 'danger', '~> 2.0', require: false
end
```

**Comment by dblock** [comment]
I opened https://github.com/danger/danger/issues/394 for Danger + JRuby, thanks @Bugagazavr.

**Pull request merged**